homeassistant/components/hydrawise/binary_sensor.py

```diff
@@ -65,5 +67,5 @@ def _update_attrs(self) -> None:
         if self.entity_description.key == "status":
             self._attr_is_on = self.coordinator.last_update_success
         elif self.entity_description.key == "is_watering":
-            zone: Zone = self.zone
+            zone = cast(Zone, self.zone)
```

Comment:
The type is already Zone | None, let's use an assert here instead

Reply:
Done